python/pytorchserver/pytorchserver/__main__.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfserving
+import argparse
+
+from pytorchserver import PyTorchModel
+
+DEFAULT_MODEL_NAME = "model"
+DEFAULT_LOCAL_MODEL_DIR = "/tmp/model"
+DEFAULT_MODEL_CLASS_NAME = ""
```

Comment:
Useless to have an empty string default.

Reply:
Done